include/argon2.h

```diff
@@ -215,7 +213,7 @@ typedef enum Argon2_type { Argon2_d = 0, Argon2_i = 1 } argon2_type;
  */
 ARGON2_PUBLIC int argon2_ctx(argon2_context *context, argon2_type type);
 
-/**
+/*
```

Comment:
The double stars (`/**`) are not a typo. They are required for tools like Doxygen to parse the comment and generate documentation out of it.